### DISCUSSION TITLE
Using Screen Broadcast Extension on background app

### DISCUSSION THREAD

**Discussion by EnterSoftwareSolutions** [flutter-webrtc/flutter-webrtc#1247] [Q&A]
Hello, 
i configured as you wrote my broadcast Extension for ios, now i can grab mobile screen extension/flutter app and send it by Webrtc connection
My problem is: when i put the flutter app in background mode (it is necessary to grab the ios screen) after 30 seconds the app stops to work. How can tell to ios to not "freeze" the app when goes in background mode? 
I try to use Background fetch or background processing but ios starts them after 30 seconds...i need to keep my app working in background...
Thanks!

**Comment by cloudwebrtc** [comment]
You may need to set the ios app to support the background, such as voip
